.ci/magician/cmd/test_terraform_vcr.go

```diff
@@ -120,7 +126,7 @@ var testTerraformVCRCmd = &cobra.Command{
 		}
 		ctlr := source.NewController(env["GOPATH"], "modular-magician", env["GITHUB_TOKEN_DOWNSTREAMS"], rnr)
 
-		vt, err := vcr.NewTester(env, rnr)
+		vt, err := vcr.NewTester(env, "ci-vcr-logs", "vcr-check-cassettes", rnr)
```

Comment:
this is inconsistent with the previous usage of vcr-check-cassettes (and it wasn't present in this file previously) - is this intentional?

Reply:
if it's an error - this should be easier to check after the PRs are split out

Reply:
It is an error, and I'm reverting it now.

Going to work on the trivial changes PR first and then squash and rebase this PR onto it.